aiozk/connection.py

```diff
@@ -258,6 +258,8 @@ def drain_all_pending(self):
     async def close(self, timeout):
         if self.closing:
             return
+        if self.read_loop_task:
+            await asyncio.wait_for(self.read_loop_task, 0.1)
```

Comment:
@cybergrind what do you think about it?

I am using `asyncio.wait_for` here because `xid, zxid, response = await self.read_response()` within `Connection.read_loop` is blocking and cannot be simply cancelled without timeout.